aten/src/ATen/core/ivalue.cpp

```diff
@@ -516,18 +517,28 @@ IValue IValue::deepcopy(
     }
       break;
     case IValue::Tag::Object: {
-      copy = IValue(toObject()->deepcopy(memo));
-      break;
+      auto class_type = type()->expect<ClassType>();
```

Comment:
what about `Capsule`?

Reply:
Capsule is only used in torchbind and should never appear here, as all Capsule instances will be attributes of an Object with `__getstate__`/`__setstate__`